frontend/src/pages/organization/SecretScanningPage/route.tsx

```diff
@@ -1,25 +1,25 @@
-import { createFileRoute, stripSearchParams } from "@tanstack/react-router";
-import { zodValidator } from "@tanstack/zod-adapter";
-import { z } from "zod";
+import { createFileRoute /* stripSearchParams */ } from "@tanstack/react-router";
 
+// import { zodValidator } from "@tanstack/zod-adapter";
+// import { z } from "zod";
 import { SecretScanningPage } from "./SecretScanningPage";
 
-const SecretScanningQueryParams = z.object({
-  state: z.string().catch(""),
-  installation_id: z.string().catch("")
-});
+// const SecretScanningQueryParams = z.object({
+//   state: z.string().catch(""),
+//   installation_id: z.string().catch("")
+// });
 
 export const Route = createFileRoute(
   "/_authenticate/_inject-org-details/organization/_layout/secret-scanning"
 )({
-  component: SecretScanningPage,
-  validateSearch: zodValidator(SecretScanningQueryParams),
-  search: {
-    middlewares: [
-      stripSearchParams({
-        installation_id: "",
-        state: ""
-      })
-    ]
-  }
+  component: SecretScanningPage
```

Comment:
@akhilmhdh Can you please advice on the correct Tanstack validation? The ones we have prevented installation_id and state to be filtered out from the redirect